Sources/ForageSDK/Foundation/Network/LDManager.swift

```diff
@@ -11,12 +11,102 @@ import LaunchDarkly
 
 internal protocol LDClientProtocol {
     func doubleVariationWrapper(forKey key: String, defaultValue: Double) -> Double
+    func jsonVariationWrapper(forKey key: String, defaultValue: LDValue) -> LDValue
 }
 
 extension LDClient: LDClientProtocol {
     func doubleVariationWrapper(forKey key: String, defaultValue: Double) -> Double {
         return self.doubleVariation(forKey: key, defaultValue: defaultValue)
     }
+    
+    func jsonVariationWrapper(forKey key: String, defaultValue: LDValue) -> LDValue {
+        return self.jsonVariation(forKey: key, defaultValue: defaultValue)
+    }
+}
+
+extension LDValue {
```

Comment:
@djoksimo let me know if you find this to be cleaner or not. You still have to convert the many layers of the `LDValue`, but at least it is more modular and feels native to the `LDValue` class now.

Reply:
Looks cleaner! I like the `extension` approach that you took. Thank you @dleis612!